File: Javascript/Lesson 23, checkpoint project/fizzbuzz.js
//fizz buzz
let number = 1; //assigning initial value
let max = 15; //assigning max value

for (let i = number; i <= max; i++) {//opening the loop, initial value, max value, and iterations
  if (i % 3 === 0 && i % 5 === 0) {//looking at multiples of 3 and 5 first
    console.log("FizzBuzz"); //print fizzbuzz if multiple of 3 and 5
  } else if (i % 3 === 0) { //next looking at multiples of 3
    console.log("Fizz"); //print fizz
  } else if (i % 5 === 0) { //looking at multiples of 5
    console.log("Buzz"); //print buzz
  } else {
    console.log(i); //if neight multiple of 3 or 5, just print the iteration number
  }
}

File: Python/Lesson 21/fizzbuzz.py
#fizz buzz problem
#needed to loop through the range from 1 to 100, iterating by 1
#needed to check for multiples of 3, or 5, and, 3 and 5
#needed to check for multiples of 3 and 5 first, otherwise they would have received the other prints
for i in range(1,100,1):
  #check for multiples of 3 and 5 by checking that the remainder of i/3 and i/5 is 0
  if i % 3 == 0 and i % 5 == 0:
    print('FizzBuzz')
    #check for multiples of 3
  elif i % 3 == 0:
    print('Fizz')
    #check for multiples of 5
  elif i % 5 == 0:
    print('Buzz')
    #if not a multiple of 3 or 5, print the number i
  else:
    print(i)
